.buildkite/pipeline.yml

```diff
@@ -178,12 +178,10 @@ steps:
           json-to-env:
             - secret-id: sdlc/prod/buildkite/terraform_provider_redpanda
       - docker#v5.11.0:
-          image: "goreleaser/goreleaser:latest"
+          image: "glrp/make-goreleaser:latest"
```

Comment:
What's `glrp`? Should this be hosted under redpanda-data in someway?

Reply:
I'd definitely like to get images moved over there. For now the dockerfile is in repo and it's a public image I hosted.